src/app/gear-power/GearPower.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react';
+import { showGearPower$ } from './gear-power';
+import Sheet from '../dim-ui/Sheet';
+import { storesSelector } from '../inventory/selectors';
+import { D2Store } from '../inventory/store-types';
+import { RootState } from '../store/reducers';
+import './GearPower.scss';
+import { useSelector } from 'react-redux';
+import { t } from 'app/i18next-t';
+import { useSubscription } from 'app/utils/hooks';
+import { maxLightItemSet } from 'app/loadout/auto-loadouts';
+import { getLight } from 'app/loadout/loadout-utils';
+import BucketIcon from 'app/dim-ui/svgs/BucketIcon';
+import BungieImage from 'app/dim-ui/BungieImage';
+import { itemPop } from 'app/dim-ui/scroll';
+import { FractionalPowerLevel } from 'app/dim-ui/FractionalPowerLevel';
+import clsx from 'clsx';
+
+export default function GearPower() {
+  const stores = useSelector<RootState, D2Store[]>((state) => storesSelector(state) as D2Store[]);
+  const [selectedStore, setSelectedStore] = useState<D2Store | undefined>();
+  const reset = () => {
+    setSelectedStore(undefined);
+  };
+
+  useSubscription(() =>
+    showGearPower$.subscribe(({ selectedStoreId }) => {
```

Comment:
This is definitely the pattern I've been using, but I kinda wish we had a lighter-weight event system for this stuff than all of RxJS. I guess the right answer is that we should just set a value in Redux state. This isn't really a comment on your code, more a "I wish this was generally different" musing.